source/install/upgrade_from_mongodb.rst

```diff
@@ -100,5 +117,35 @@ Upgrading on Red Hat Enterprise Linux or CentOS
    example, existing data may not be compatible with the default WiredTiger
    storage engine.
 
+Upgrading a replica set
+================================================================================
+
+The :term:`rolling restart <Rolling restart>` method lets you upgrade your replica set to |PSMDB| with minimum downtime. You upgrade the nodes one by one while the whole cluster remains operational.   
+
+Upgrade the replica set secondary nodes  
+--------------------------------------------------------------------------------
+
+1. Upgrade a secondary node in a replica set as described in :ref:`upgrade_standalone`. Use the instructions relevant to your operating system. 
+#. Wait for the node to rejoin with the replica set members and report the SECONDARY status. 
+#. Repeat the upgrade procedure on the remaining secondary nodes. 
+   
+Upgrade the primary node
+--------------------------------------------------------------------------------
+
+1. Step down the primary node: :command:`rs.stepDown()`.
+2. Wait for the remaining nodes to elect a new primary. Run :command:`rs.status()` to verify that the former primary node reports as SECONDARY.
+#. Upgrade the node as described in :ref:`upgrade_standalone`.
+   
+.. seealso::
+
+   |mongodb| Documentation: Upgrade a Replica Set
+       https://docs.mongodb.com/manual/release-notes/4.2-upgrade-replica-set/
+
+.. note::
+
+   Steps to upgrade from |mongodb-ce| with data encryption enabled to |PSMDB| are different. ``mongod`` requires an empty ``dbPath`` data directory because it cannot encrypt data files in place. It must receive data from other replica set members during the initial sync. Please refer to the :ref:`switch_storage_engines` for more information on migration of encrypted data. `Contact us <https://www.percona.com/about-percona/contact#us>`_ for working at the detailed migration steps, if further assistance is needed.
```

Comment:
Same is also true for situations when user wants to move from unencrypted storage to encrypted one or vice versa. Just my thoughts.

Reply:
> Same is also true for situations when user wants to move from unencrypted storage to encrypted one or vice versa.

I think they'll get the implication.